css/fashion_main_styles.css

```
@@ -23,23 +25,225 @@
   position: absolute;
   bottom: 0;
   width: 40%;
+  padding-bottom: 20px;
+}
+.poster-text h1 {
+  font-weight: bold;
+  font-size: 26px;
 }
 
-.poster-text-small {
-  font-family: PT Serif;
+.wide-post-container {
+  text-align: left;
+  padding: 25px 126px 25px 30px;
+}
+
+.post-item-container {
+  padding: 15px 14px 14px 14px;
 }
 
 .font-article-section {
   text-transform: uppercase;
   font-family: PT Sans;
```

`font-family` needs a generic default font in case main is not available to a browser
```suggestion
  font-family: PT Sans, sans-serif;
```